catalog/src/main/java/it/eng/catalog/exceptions/CatalogExceptionAdvice.java

```diff
@@ -2,44 +2,40 @@
 
 import java.util.Arrays;
 
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 import it.eng.catalog.model.CatalogError;
-import it.eng.catalog.model.GenericApiResponse;
 import it.eng.catalog.model.Reason;
 import it.eng.catalog.model.Serializer;
 import it.eng.catalog.rest.protocol.CatalogProtocolController;
-import it.eng.catalog.service.CatalogService;
 import jakarta.validation.ValidationException;
 
-@RestControllerAdvice(basePackageClasses = {CatalogProtocolController.class, CatalogService.class})
+@RestControllerAdvice(basePackageClasses = CatalogProtocolController.class)
```

Comment:
With excluding Protocol from names in classes, should we do the same with Controller?
And use API in all other cases where non-protocol code is involved.

Reply:
Done 👍